client/client.go

```diff
@@ -191,8 +191,11 @@ func (cli *Client) Close() error {
 
 // getAPIPath returns the versioned request path to call the api.
 // It appends the query parameters to the path if they are not empty.
-func (cli *Client) getAPIPath(p string, query url.Values) string {
+func (cli *Client) getAPIPath(p string, query url.Values) (string, error) {
 	var apiPath string
+	if strings.Contains(p, "../") {
```

Comment:
I'm not sure this is correct since this would also ban `foo../` which isn't what we want.
I know we may not ever have "foo.." as a valid part of the URL but I think that's a different topic. Perhaps changing it to "/../" if we really want to do this.